modules/ROOT/pages/_partials/keycloak/5_binding-an-app-to-a-service.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+// Base the file name and the ID on the module title. For example:
+// * file name: doing-procedure-a.adoc
+// * ID: [id='doing-procedure-a']
+// * Title: = Doing procedure A
+
+// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+:context: {keycloak-service}
+[id='binding-an-app-to-{context}']
+// The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
+= Binding a {mobile-client} with the {keycloak-service} Service
+
+To use mobile services, you must represent your mobile app in *Mobile Developer Console*, and that app must be associated with the mobile service.
+This association is called *binding* and it is necessary for your mobile app to use that service.
+
+This section describes how to set up the {keycloak-service} service.
+
+.Prerequisites
+
+* xref:getting-started-with-{context}[Getting Started with the {keycloak-service} Service].
```

Comment:
@jstaffor I think Prereqs formulated like this (ie links) aren't that useful. The don't really follow the spirit of minimalism and/or mod docs, perhaps no links and the 'Success criteria' from the trello column would be better.

Reply:
Can you provide me with an example as to how you would do it?

Reply:
looking back at trello, we have a 'prerequisites' column, so that would point us to having 'prereqs'  at the assembly level, and that covers all you need to do for the whole IDM story. Then we don't need prereqs for the individual 'modules' and this makes sense as we won't have any reuse possibility, and users can't navigate to a module without the context of the bigger story. 
We can revisit later if necessary

Reply:
- re "we won't have any reuse possibility" - we may not but other CCS folks may in the future...it's worth considering this point.

- 'prereqs' at the assembly level - that makes sense. I can see how this is more minimalistic. 

In summary, I think it's best to revisit at a later point for sure - once the full picture becomes clearer. When the full picture becomes clear - I think that this will be a good point to trash out such things as minimalism / mod docs / etc.